src/shared/identityUtils.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+
+import { randomBytes } from 'node:crypto';
+import { DevServerUtils } from './devServerUtils.js';
+
+class LwrConfigFile {
+  public identityToken?: string;
+}
+
+// **********************************************************************************************
+// * TODO: When we finalize the implementation for the preview commands and things settle down, *
+// *       consider moving this class into CryptoUtils of lwc-dev-mobile-core instead.          *
+// **********************************************************************************************
+export class IdentityUtils {
+  public static async updateServerConfigFileWithIdentityToken(byteSize = 32): Promise<void> {
+    const config = await DevServerUtils.fetchServerConfigFileContent() as LwrConfigFile;
+    if (config && !config.identityToken) {
+      config.identityToken = randomBytes(byteSize).toString();
```

Comment:
This didn't occur to me earlier until now. `randomByte` can return bytes that don't represent 'normal' characters so I don't think we can just convert to string and write to file. Based on documentation from [here](https://nodejs.org/api/crypto.html#cryptorandombytessize-callback) we should at least use `.toString('hex')`

Not sure how this will impact the server parsing this value though.

Reply:
Aah, great catch. I actually noticed this when I was testing last week and thought I changed it to 'hex'. Let me put it back.

Reply:
A few thoughts here:

- In the context of [my previous comment](https://github.com/salesforcecli/plugin-lightning-dev/pull/52/files#r1636897993), the heart of this should move into its own method in `CryptoUtils` in `lwc-dev-mobile-core`.
- This can simply be a function that creates and returns a random token value, based on an input number of bytes. Default could be `32`.
- The return value should be a base64-encoded string, to align with the other areas where we'll use base64 encoding for these values. So i.e. return `randomBytes().toString('base64')`.